src/DynamicHTML.js

```diff
@@ -1,8 +1,9 @@
 export const populateScores = (scoresArray) => {
   const scoresTable = document.getElementById('scores_table');
-  scoresTable.innerHTML = '<th><tr><td>Player</td><td>Score</td></tr></th>';
+  scoresTable.innerHTML = '<th><tr style="color:Orange;"><td>Player</td><td>Score</td></tr></th>';
   for (let i = 0; i < scoresArray.length; i += 1) {
     const tr = document.createElement('tr');
+    tr.classList.add('rowStyle');
     const td1 = document.createElement('td');
     const td2 = document.createElement('td');
     td1.append(`${scoresArray[i].user}`);
```

Comment:
- [ ] I'm facing this error message when I run your application on my computer, Kindly fix this 👇🏻 

![21](https://user-images.githubusercontent.com/79058364/180308587-b224712b-60a0-4246-9ecd-5706d0360aa7.png)